### PR TITLE
update the benchmark specifications

### PR DESCRIPTION
**Pull request by Trisfald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,12 @@ Contributions are most welcome!
...
 
-That being said, the service implementations should **NOT** take advantage of that and keep the code generic and maintainable. No inline assembly or other, language specific tricks / hacks. What does generic mean? One should be able to easily adapt the existing code to some fundamental use cases (e.g. having a thread-safe hash map on server side to provide values to client given some key).
+That being said, the service implementations should **NOT** take advantage of that and keep the code generic and maintainable. What does generic mean? One should be able to easily adapt the existing code to some fundamental use cases (e.g. having a thread-safe hash map on server side to provide values to client given some key or performing blocking I/O).\
+Keep in mind the following guidelines:
+- No inline assembly or other, language specific, tricks / hacks should be used
+- The code should be (reasonably) ideomatic, built upon the modern patterns of the language
```
Comment (LesnyRumcajs) [inline README.md:14]: idiomatic?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,12 @@ Contributions are most welcome!
...
+- Don't make any assumption on the kind of work done inside the server's request handle
```
Comment (LesnyRumcajs) [inline README.md:15]: handler?

File: README.md [hunk 3]
```diff
@@ -8,7 +8,12 @@ Contributions are most welcome!
 
 The goal of this benchmark is to compare the performance and resource usage of various gRPC libraries across different programming languages and technologies. To achieve that, a minimal protobuf contract is used to not pollute the results with other concepts (e.g. performances of hash maps) and to make the implementations simple.
 
-That being said, the service implementations should **NOT** take advantage of that and keep the code generic and maintainable. No inline assembly or other, language specific tricks / hacks. What does generic mean? One should be able to easily adapt the existing code to some fundamental use cases (e.g. having a thread-safe hash map on server side to provide values to client given some key).
+That being said, the service implementations should **NOT** take advantage of that and keep the code generic and maintainable. What does generic mean? One should be able to easily adapt the existing code to some fundamental use cases (e.g. having a thread-safe hash map on server side to provide values to client given some key or performing blocking I/O).\
```
Comment (LesnyRumcajs) [inline README.md:11]: maybe grabbing a network resource would be a more straightforward use case?

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Trisfald) [reply README.md:14]: of course

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**